docker/Dockerfile

```diff
@@ -10,6 +10,6 @@ FROM alpine:edge
 RUN apk add --no-cache bash
 SHELL ["/bin/bash", "-c"]
 COPY --from=builder /app/app ./
-COPY ./docker/entrypoint.sh /entrypoint.sh
+COPY ./docker/docker-entrypoint.sh /entrypoint.sh
 
 ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
entrypoint.sh used twice above but but the file is deleted

Reply:
in line 13 we copy the file `docker-entrypoint.sh` from the `docker` dir to the image and call it `entrypoint.sh` there

Reply:
oh my bad, I thought we copying two different files there